rogueutil.h

```diff
@@ -613,6 +613,8 @@ setString(const RUTIL_STRING & str_)
 void
 setString(RUTIL_STRING str)
 {
+	char buf[3 + 20 + 1]; /* 20 = max length of 64-bit
```

Comment:
buf is unused in the _WIN32 block and thus it would make no sense for it to be declared there, as it would probably be optimized away.

Reply:
I'm not quite sure what you mean here. buf is not declared in the _WIN32 block, it is declared in the ```#else /* __cplusplus */``` block.

Reply:
You are right, I meant the `__cplusplus` block, but `_WIN32` is somewhat relevant. My point was that it is unused in any of the conditions leading up to the else block.
If `__cplusplus` is defined, but `buf` was unconditionally declared earlier, then `buf` would be a dangling unused variable and would likely fail compilation if `-Werror` is active.

You could surround the declaration with another `ifdef`, but that might be too messy. Considering what the code looks like after the preprocessor, I think the declaration is stylistically fine where it was.